src/pages/Gyms.js

```diff
@@ -37,15 +38,14 @@ const defaultFilters = {
   six: false,
   twelve: false,
   twentyfour: false,
-  price: 10
+  price: 90
```

Comment:
added price to the state which was moved down to the PriceBar

Reply:
Regarding: "was moved down...", you probably mean "is passed on to... component"